interface/controller/item_controller.go

```diff
@@ -0,0 +1,54 @@
+package controller
+
+import (
+	"errors"
+	"net/http"
+	"strconv"
+
+	"academy_bootcamp/application/usecase"
+	"academy_bootcamp/domain/model"
+)
+
+type itemController struct {
+	Usecase usecase.ItemService
+}
+
+type ItemController interface {
+	GetItemById(c Context) error
+	// CreateItem(c Context) error
+}
+
+func NewItemController(us usecase.ItemService) ItemController {
+	return &itemController{us}
+}
+
+func (uc *itemController) GetItemById(c Context) error {
```

Comment:
For better readability, acronyms should be declared in all uppercases.
```suggestion
func (uc *itemController) GetItemByID(c Context) error {
```